storm-client/src/storm.thrift

```diff
@@ -477,12 +477,13 @@ struct SupervisorInfo {
     1: required i64 time_secs;
     2: required string hostname;
     3: optional string assignment_id;
-    4: optional list<i64> used_ports;
-    5: optional list<i64> meta;
-    6: optional map<string, string> scheduler_meta;
-    7: optional i64 uptime_secs;
-    8: optional string version;
-    9: optional map<string, double> resources_map;
+    4: optional i32 server_port;
```

Comment:
-1 we cannot change the tags for a thrift object like this.  It totally breaks backwards compatibility.  Granted this is for an internal class and probably does not matter much, but there is no reason to do it.  Please make the server_port 10: and set all of the others back to where they were before.